dashboard-ui/src/pages/home.tsx

```diff
@@ -83,9 +99,18 @@ function getContainerIDs(
     if (ownershipMap.has(childID)) getContainerIDs(childID, ownershipMap, containerIDs);
     else containerIDs.push(childID);
   });
+
   return containerIDs;
 }
 
+/**
+ * Checks if all provided arrays are either undefined or empty
+ */
+
+export function noSearchResults(...arrays: (WorkloadItem[] | undefined)[]) {
```

Comment:
React hot module replacement isn't working currently because apparently it only works if all exported objects are react components. To fix this we should move the helpers to a library method (e.g. `/src/lib/home.ts`) then that should fix the problem.